### PR TITLE
Refactor [v100] FXIOS-3906 - Telemetry improvement added top and bottom toolbar location for cfr event

### PR DESCRIPTION
**Pull request by nbhasin2**

### CODE REVIEW

File: Client/Frontend/Contextual Hint/ContextualHintViewModel.swift [hunk 1]
```diff
@@ -16,6 +16,10 @@ enum ContextualHintViewType: String {
     case jumpBackIn = "JumpBackIn"
     case inactiveTabs = "InactiveTabs"
     case toolbarLocation = "ToolbarLocation"
+
+    var hintExtraForToolbarLocation: String {
+        BrowserViewController.foregroundBVC().isBottomSearchBar ? "ToolbarLocationBottom" : "ToolbarLocationBottom"
```
Comment (electricRGB) [inline Client/Frontend/Contextual Hint/ContextualHintViewModel.swift:21]: uhhhhhh, they're both bottom? :P

```suggestion
        BrowserViewController.foregroundBVC().isBottomSearchBar ? "ToolbarLocationBottom" : "ToolbarLocationTop"
```

**Review by electricRGB — APPROVED** [review]
Gotta make that one change but good otherwise

**Comment by nbhasin2** [comment]
# Request for data collection review form

All questions are mandatory. You must receive review from a data steward peer on your responses to these questions before shipping new data collection.

1. What questions will you answer with this data?
-  How many users have their toolbar set to bottom or top.

2. Why does Mozilla need to answer these questions? Are there benefits for users? Do we need this information to address product or business requirements?
  * This will help us understand usage patterns around the awesomebar location preference 
  
3. What alternative methods did you consider to answer these questions? Why were they not sufficient?
  * There are no other alternatives.
 
4. Can current instrumentation answer these questions?
  * No


5. List all proposed measurements and indicate the category of data collection for each measurement, using the Firefox data collection categories found on the Mozilla wiki.
Note that the data steward reviewing your request will characterize your data collection based on the highest (and most sensitive) category.

Measurement Description | Data Collection Category | Tracking Bug #
-- | -- | --
Record awesome bar shown with location top or bottom on app stratup | Category 2 - interaction data | https://mozilla-hub.atlassian.net/browse/FXIOS-3906

6. Please provide a link to the documentation for this data collection which describes the ultimate data set in a public, complete, and accurate way.
  * https://dictionary.telemetry.mozilla.org/apps/firefox_ios

7. How long will this data be collected? Choose one of the following:
  * This will expire on 2022-09-01

8. What populations will you measure?
  * All channels, all locales, all countries

9. If this data collection is default on, what is the opt-out mechanism for users?
  * Default Glean SDK opt-out mechanism.

10. Please provide a general description of how you will analyze this data.
  * Glean

11. Where do you intend to share the results of your analysis?
  * Only on Glean, and with mobile teams.

12. Is there a third-party tool (i.e. not Telemetry) that you are proposing to use for this data collection?
   * No third-party tools.

**Review by electricRGB — APPROVED** [review]

**Comment by dnarcese** [comment]
r+
# Data Review Form

_1) Is there or will there be **documentation** that describes the schema for the ultimate data set in a public, complete, and accurate way?_ 

Yes, telemetry is documented at https://dictionary.telemetry.mozilla.org/apps/firefox_ios

_2) Is there a control mechanism that allows the user to turn the data collection on and off?_ 

Yes, Firefox iOS provides data controls in the app settings

_3) If the request is for permanent data collection, is there someone who will monitor the data over time?_

This should be re-reviewed w/ Ashley Thomas every 6mo

_4) Using the **[category system of data types](https://wiki.mozilla.org/Firefox/Data_Collection)** on the Mozilla wiki, what collection type of data do the requested measurements fall under?_

Type 2

_5) Is the data collection request for default-on or default-off?_

Default on

_6) Does the instrumentation include the addition of **any *new* identifiers** (whether anonymous or otherwise; e.g., username, random IDs, etc.  See the appendix for more details)?_

No

_7) Is the data collection covered by the existing Firefox privacy notice?_ 

Yes

_8) Does the data collection use a third-party collection tool?_ 

No

**Pull request merged**